source/features/command-palette-navigation-shortcuts.tsx

```diff
@@ -0,0 +1,32 @@
+import select from 'select-dom';
+import onetime from 'onetime';
+
+import features from '.';
+
+function commandPaletteKeydown({key, ctrlKey, currentTarget}: KeyboardEvent): void {
+	if (!currentTarget || !ctrlKey || (key !== 'n' && key !== 'p')) {
+		return;
+	}
+
+	const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
+	currentTarget.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
+}
+
+function init(): void {
+	const commandPalette = select('command-palette');
+
+	if (!commandPalette) {
+		return;
+	}
+
+	commandPalette.addEventListener('keydown', commandPaletteKeydown);
```

Comment:
This is the perfect candidate for delegation:

```suggestion
	delegate(document, 'command-palette', 'keydown', commandPaletteKeydown);
```